### ISSUE TITLE
problem with dependencies when make sphinx target

### ISSUE THREAD

**Issue by rrsettgast** [LLNL/blt#217]
Sometimes when running the sphinx make target generated by blt, modified dependencies appear to miss being reprocessed by sphinx. My basis for this statement is that I can be modifying documentation and running the make target and everything is fine...but if I rerun cmake to regenerate the build directory, I will get errors that I didn't see from previously. Perhaps an option to do a full rebuild would solve this?

https://stackoverflow.com/questions/21019505/sphinx-force-rebuild-of-html-including-autodoc

**Comment by cyrush** [comment]
correct, our current sphinx builds are incremental. I am open to doing a force rebuild -- what do other folks think? @kennyweiss @white238 ?

**Comment by kennyweiss** [comment]
I haven't played around with it enough to have a strong opinion. 

Based on the discussion, it seems like rebuilding would be preferable for repeat builds when one is actively working on the docs (at some additional generation costs), and would likely not have much of an impact when one is not actively working on the docs.  If so, my vote is for rebuilding by default.
